.github/sync.yml

```diff
@@ -1,7 +1,25 @@
 group:
   - repos: |
+      conda/actions
+      conda/ceps
+      conda/clabot-config
       conda/conda
+      conda/conda-benchmarks
+      conda/conda-bots
       conda/conda-build
+      conda/conda-content-trust
+      conda/conda-docs
+      conda/conda-pack
+      conda/conda-package-handling
+      conda/conda-prefix-replacement
+      conda/conda-verify
+      conda/conda_build_test_recipe
```

Comment:
Don't include conda_build_test_recipe since we will transition this into an archived repo